README.md

```diff
@@ -53,6 +57,7 @@ example, a request to `/foo/baz` made over HTTP will result in a request to
 `http://example.com/bar/baz`.
 
 You can also specify the scheme or choose a port:
+
```

Comment:
There have happened some formatter changes here as well. I can roll those back if you see fit.